utils/src/main/java/org/nervos/ckb/crypto/Hash.java

```diff
@@ -0,0 +1,59 @@
+package org.nervos.ckb.crypto;
+
+import org.bouncycastle.jcajce.provider.digest.Keccak;
+import org.nervos.ckb.utils.Numeric;
+
+import java.nio.charset.StandardCharsets;
+
+/**
+ * Cryptographic hash functions.
+ */
+public class Hash {
+    private Hash() { }
+
+    /**
+     * Keccak-256 hash function.
+     *
+     * @param hexInput hex encoded input data with optional 0x prefix
+     * @return hash value as hex encoded string
+     */
+    public static String sha3(String hexInput) {
```

Comment:
Is this the standard SHA3 (256)? Currently CKB and other SDKs only use standard SHA3, not Keccak256.